hoot-core/src/main/cpp/hoot/core/io/BulkInsert.cpp

```diff
@@ -22,6 +22,6 @@
  * This will properly maintain the copyright information. DigitalGlobe
  * copyrights will be updated automatically.
  *
- * @copyright Copyright (C) 2015 DigitalGlobe (http://www.digitalglobe.com/)
+ * @copyright Copyright (C) 2016 DigitalGlobe (http://www.digitalglobe.com/)
```

Comment:
Yeah I'm with @demmith on this.  The script shouldn't be destructive.  Those years are in there for a reason and if the git history is gone for those years we should still honor what is there.